collections/guides/use_token_data_in_http_requests/index.md

```diff
@@ -0,0 +1,152 @@
+categories: guides
+nav_order: 5
+has_children: true
+has_toc: false
+image:
+    path: https://cdn.basistheory.com/images/seo/guides-opengraph.png
+    width: 1200
+    height: 630
+---
+
+# Use Token Data in HTTP Requests
+{: .no_toc }
+
+In this guide, we will walk through how to use the Proxy to send an outbound HTTP request containing sensitive detokenized data.
+
+We will be leveraging the third party service https://httpbin.org/ as our external destination API. 
+HTTPBin is a free service that will accept any input request and respond with information about the request it received. 
+This will allow us to inspect the manipulated request that was forwarded from the Proxy to the destination API.
+
+If you would like to learn more about the Basis Theory Proxy first, or how this product can help you meet your security and compliance needs, check out [What is the Proxy?](/concepts/what-is-the-proxy) before continuing.
+
+### Table of contents
+{: .no_toc .text-delta }
+1. 
+{:toc}
+
+## Prerequisites
+
+### Application Setup
+
+We will be creating and proxying tokens that contain PCI data in this example.
+First, we need to ensure we have a `Server to Server` application configured that has `token:pci:create` and `token:pci:use:proxy` permissions.
+We will be using the API Key associated with this application in all HTTP requests below.
+
+<span class="base-alert info">
+  <span>
+    The Proxy requires the `token:<classification>:use:proxy` permission for each classification of token that you wish to include in the proxy request.
+    For more information about Proxy permissions, see <a href="https://docs.basistheory.com/api-reference/#permissions-permission-types">our docs</a>.
+  </span>
+</span>
+
+### Create a Token
+
+Next, we will create a `card_number` token containing a credit card number we wish to secure with Basis Theory.
+
+```js
+    curl "https://api.basistheory.com/tokens" \
+      -H "BT-API-KEY: key_NS21v84n7epsSc5WzoFjM6" \
+      -H "Content-Type: application/json" \
+      -X "POST" \
+      -d '{
+            "type": "card_number",
+            "data": "6011111111111117"
+        }'
+```
+
+This responds with the created token:
+
+```js
+{
+    "id": "a70f8701-8b65-476b-ac0a-21bcd7d943d5",
+    "type": "card_number",
+    "data": "XXXXXXXXXXXX1117",
+    ...
+}
+```
+
+We will be using the id of this token below to build a request to the Proxy.
+
+## Create the Proxy Request
+
+In this example, we will be working with a hypothetical third party HTTP API that requires credit card information as input. 
+This API will be available at `https://httpbin.org/anything` and it expects an HTTP POST request with a request body of the form:
+```js
+{
+  "firstName": <string>,
+  "lastName": <string>,
+  "creditCardNumber": <string>,
+  "expirationDate": <string>
+}
+```
+
+Our hypothetical source system will be storing plaintext first and last names and expiration dates of card numbers, as these fields are considered non-sensitive.
```

Comment:
plaintext first names, last names and expiration dates? idk the double and sounded weird